executors/sshexec/volume.go

```diff
@@ -51,23 +49,18 @@ func (s *SshExecutor) VolumeCreate(host string,
 		maxPerSet = 1
 	}
 
-	// Setup volume create command
 	// There could many, many bricks which could make the command line
 	// too long.  Instead, create the volume first, then add each brick set.
 	for _, brick := range volume.Bricks[:inSet] {
 		cmd += fmt.Sprintf("%v:%v ", brick.Host, brick.Path)
 	}
 
-	// Initialize the commands with the create command
 	commands := []string{cmd}
 
-	// Now add all the commands to add the bricks
```

Comment:
I request this one line to stay as a continuation of comments on lines 55,56. I know it is self explanatory but it is a "different way of doing things" and would be good to have a comment to get attention of developer.